client/testClient.py

```diff
@@ -51,8 +51,28 @@ def _json_decode(json_bytes, encoding):
 if content_length == 0:
     print("***No updates from server***")
 else:
+    content_length = int(content_length)
     content = sock.recv(content_length)
-    content = _json_decode(content, "utf-8")
-    print(content)
+    print(len(content))
+    print(content_length)
+    # content = _json_decode(content, "utf-8")
+    filename = "client_executable.exe"
```

Comment:
Writing to file this way will cause problems when the size of the incoming file is large, content will not be able to hold a string 100+ MBs